contrib/lkt/extensions/mains/lkt_toolbox.adb

```diff
@@ -90,6 +90,8 @@ begin
             Unit : constant Analysis.Analysis_Unit
               := Ctx.Get_From_File (To_String (File_Name));
          begin
+            Unit.Populate_Lexical_Env;
```

Comment:
I would add a comment explaining why we explicitly call `Populate_Lexical_Env` here, instead of letting it be done automatically during the call to the `P_Check_Semantic` property.

Reply:
Done